broker/pkg/carrier/loader.go

```diff
@@ -189,14 +190,18 @@ func (c *Loader) hydrateOneConsumer(ctx context.Context, limit int, tuple *Consu
 	for hasMoreForThisBucket {
 		pushLeaseMessages, hasMoreForThisBucket, err = c.storage.SelectByConsumer(ctx, tuple.C.GetTopicAsBytes(), tuple.C.GetIDAsBytes(), fullRange, consumerLimit, dtime.TimeToMS(time.Now())+c.conf.PrefetchMaxMilliseconds)
 		if err != nil {
+			metricTopicLeasesErrors.With(prometheus.Labels{"operation": "create", "topic": tuple.C.GetTopic()}).Inc()
```

Comment:
It should be incremented with the number of leases that had this error

Reply:
we should add another metric in the storage for that
and rename this one because the storage only returns one error
but the storage should return after the first error so there should not be more than one